docs/index.md

```diff
@@ -4103,6 +4105,109 @@ return api.getLedger()
 ```
 
 
+## parseAccountFlags
+
+`parseAccountFlags(Flags: number): object`
+
+Parse an `AccountRoot` object's [`Flags`](https://developers.ripple.com/accountroot.html#accountroot-flags).
+
+### Parameters
+
+This method takes one parameter, the `Flags` number to parse.
+
+### Return Value
+
+This method returns an object with containing a key for each parsed flag. Each flag has a boolean value of `true` or `false`.
```

Comment:
The example implies that flags that are `false` are omitted from the return value, or at least that flags whose value is their default value are omitted. That should be explicit. Alternatively, it would be OK (maybe more useful?) if all known flags were included (even if most of them are `false`).